spec/feature_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'takeaway'
+
+describe 'User stories' do
+
+  #1 User Story
+  #As a customer
+  #So that I can check if I want to order something
+  #I would like to see a list of dishes with prices
+  context 'to see a menu' do
+    it 'returns a list of dishes with prices ' do
+    takeaway = Takeaway.new(Menu.new, Messager.new)
+      expect(takeaway.display_menu).to eq ({"chicken" =>3,
+     "rice" =>1,
+     "pizza" =>6,
+     "chips" =>2 })
+    end
+  end
+
+  #2 User Story
+  #As a customer
+  #So that I can order the meal I want
+  #I would like to be able to select some number of several available dishes
+  context 'to place an order' do
+    it 'adds an item to the basket' do
+      takeaway = Takeaway.new(Menu.new, Messager.new)
+      expect(takeaway.basket.size).to eq(0)
+      takeaway.select_dishes("chips", 2)
+      expect(takeaway.basket.size).to eq(1)
```

Comment:
Consider law of demeter - this property belongs to the order class - https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#law-of-demeter.  Also try to only use 1 expect per it block in your tests